x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -560,6 +560,10 @@ export type AlertsTableProps = {
    * Allows to consumers of the table to decide to highlight a row based on the current alert.
    */
   shouldHighlightRow?: (alert: Alert) => boolean;
+  /**
+   * Enable when rows may have variable heights (disables virtualization)
```

Comment:
[attaching to a semi random line for threading]

Are there any plans on limiting the potential number of rows per page for non-virtualized data grids? It's likely that you're going to run into major lag/perf issues rendering 1000 rows at once, for example 😅

Reply:
As of today, we can only show a maximum of 100 alert in the table

Reply:
👍 If you run into performance issues with large grids and 100 rows, it may be worth considering going down to a max of 50.